_includes/nav-bar.html

```diff
@@ -71,6 +71,7 @@
               <a class="dropdown-item" href="/events/world-moveit-day/2020/04/28/world-moveit-day-2020.html">World MoveIt Day</a>
               <a class="dropdown-item" href="/documentation/contributing/pullrequests/">Pull Requests</a>
               <a class="dropdown-item" href="/documentation/contributing/code/">Code Style</a>
+              <a class="dropdown-item" href="/documentation/contributing/releases/">Patch Releases</a>
```

Comment:
Why "Patch"? Doesn't this cover major and minor releases also? Would prefer simply "Release Process"

I also don't think we need this be on the main navigation menu, most users don't care about this level of detail. Could we just link to it from its current location on the Pull Requests page, or even better, add a left hand navigation bar in the same style as our [install page](https://moveit.ros.org/install/)?

Reply:
It definitely always was in the wrong place in "Pull Requests".
Not sure it makes sense for the website redesign as part of this documentation effort, so I would propose to merge this here and have the the layout / menu structure reworked in a later pr

Reply:
Keeping the restructure in separate PRs makes sense

Reply:
done